contained-innersource-enables-collaborative-product-development.md

```diff
@@ -0,0 +1,35 @@
+## Title  
+Contained inner source enables collaborative product development
+
+## Context
+* Multiple business units need a software component.
+* It makes sense for the business units to collaborate; each needs some different changes to the code.
```

Comment:
Btw., Tim, just like in Slack, you can use Emojis to signal you _Okay_. Tends to keep the comment stream leaner.